.vscode/launch.json

```diff
@@ -16,7 +17,16 @@
             "program": "${workspaceFolder:sourcekit-lsp}/.build/release/sourcekit-lsp",
             "args": [],
             "cwd": "${workspaceFolder:sourcekit-lsp}",
-            "preLaunchTask": "swift: Build Release sourcekit-lsp"
+            "preLaunchTask": "swift: Build Release sourcekit-lsp",
+            "sourceLanguages": ["swift"]
+        },
+        {
+            "type": "lldb",
+            "request": "attach",
+            "name": "Attach sourcekit-lsp",
+            "program": "${workspaceFolder:sourcekit-lsp}/.build/debug/sourcekit-lsp",
```

Comment:
Just a thought: Since you are hard-coding the debug configuration, should the name of this action be “Attach sourcekit-lsp (debug)” or “Attach to debug build of sourcekit-lsp”. Just so nobody gets confused if it’s not attaching properly because you’re running a release build of sourcekit-lsp.

Reply:
Fair point. I'll add it.